apps/chat/src/components/Common/ReplaceConfirmationModal/Components.tsx

```diff
@@ -193,8 +193,8 @@ const PromptView = ({ item: prompt }: PromptViewProps) => {
       <IconBulb size={18} className="text-secondary" />
       <Tooltip
         tooltip={prompt.name}
-        contentClassName="max-w-[400px]"
-        triggerClassName="truncate text-center w-full"
+        contentClassName="sm:max-w-[400px] max-w-[250px] break-all"
+        triggerClassName="truncate whitespace-pre"
       >
         <div className="truncate whitespace-pre break-all text-left">
```

Comment:
why do we have `truncate` twice for trigger?

Reply:
I removed the div